python/python/lancedb/table.py

```diff
@@ -1409,7 +1411,13 @@ def search(
             vector and the returned vector.
         """
         if vector_column_name is None and query is not None:
-            vector_column_name = inf_vector_column_query(self.schema)
+            if query_type == "fts":
+                try:
+                    vector_column_name = inf_vector_column_query(self.schema)
+                except ValueError:
+                    vector_column_name = ""
+            else:
+                vector_column_name = inf_vector_column_query(self.schema)
```

Comment:
Could you add a unit test for #1124?

Also if query type is `"fts"`, why do we even need a vector column? Does it sometime imply there is also vector search at the same time?